.travis.yml

```diff
@@ -3,12 +3,12 @@ sudo: false
 
 matrix:
   include:
-    - rust: 1.28.0
-      env: DESCRIPTION="Linux, 1.28.0"
+    - rust: 1.32.0
+      env: DESCRIPTION="Linux, 1.32.0"
       os: linux
 
-    - rust: 1.28.0
-      env: DESCRIPTION="OSX, 1.22.0"
+    - rust: 1.32.0
+      env: DESCRIPTION="OSX, 1.32.0"
```

Comment:
Is there any reason not to use 1.31 as the MSRV? Also you need to update the note in the README.

Reply:
`extern crate std;` and `use std::<..>;` in modules does not work on 1.31.0. It fails with "error[E0658]: imports can only refer to extern crate names passed with `--extern` on stable channel (see issue #53130)".

Reply:
Fair enough. 1.32 is barely over two months old, but there doesn't appear to be much demand for back compatibility. Perhaps we should not do the same for the main Rand project yet though just in case we need to revert this.